XiEditor/HoverViewController.swift

```diff
@@ -121,6 +121,7 @@ extension EditViewController {
             let positioningSize = CGSize(width: 1, height: 1) // Generic size to center popover on cursor
 
             infoPopover.show(relativeTo: NSRect(origin: positioningPoint, size: positioningSize), of: self.view, preferredEdge: .minY)
+            hoverRequestID += 1
```

Comment:
Doesn't this mean multiple requests could be sent with the same ID?

Reply:
Yes, but I'm not quite sure what the request ID is meant to represent in the first place. As it is right now, the ID doesn't get incremented if a result wasn't shown, which happens when:

1. The result we get back is empty
2. The request ID in the RPC notification we get from core is different from what's currently expected

Since (in my personal opinion) I think that hover requests should return results instantly, it's fine even if we send many requests with the same ID?